test/edu/stanford/dlss/wowza/TestSulWowza.java

```diff
@@ -738,6 +738,45 @@ public void getVerifyStacksTokenUrl_validUrl()
     }
 
     @Test
+    public void verifyTokenAgainstStacksService_filenameNeedsEncoding()
+    {
+        WMSProperties mockProperties = mock(WMSProperties.class);
+        when(mockProperties.getPropertyStr(anyString(), anyString())).thenReturn(SulWowza.DEFAULT_STACKS_TOKEN_VERIFICATION_BASEURL);
+        IApplicationInstance appInstanceMock = mock(IApplicationInstance.class);
+        when(appInstanceMock.getProperties()).thenReturn(mockProperties);
+        testModule.onAppStart(appInstanceMock);
+        String druid = "oo000oo0000";
+        String filename = "{([special-chars])}: ü@?;=&#$%20^*.|-_+!,~'\"`";
+        String userIp = "0.0.0.0";
+        String expPath = "/media/" + druid + "/" + SulWowza.urlEncode(filename) + "/verify_token";
```

Comment:
it turns out this is a pain to test, because `urlEncode` is a static method, and Mockito doesn't easily allow for verify calls to static methods.  the approach of using `spyModule = spy(testModule)` to watch `testModule` doesn't work, because you can't set an expectation for `urlEncoding` to be called on `spyModule`, because it's a static method, and thus called on the class `SulWowza`.

the [PowerMock library](https://github.com/jayway/powermock/) would seem to offer a way out of this, but in my initial testing, it was difficult to setup, and i was unable to get it to verify calls to static methods in the hour or so i played with it.  it was prone to breaking existing tests when setup improperly, and i found it a bit cumbersome and messily documented:
https://github.com/jayway/powermock/wiki/MockitoUsage#a-full-example-for-mocking-stubbing--verifying-static-method

anyway, discussed w/ @ndushay, and she agreed that it's reasonable to leave the `urlEncode` method static, and that it's reasonable to just go with the tests as-is, since the call is implicitly tested in other test cases.  would be nice to test this more thoroughly, but doesn't seem practical at the moment.